app/scripts/modules/amazon/src/loadBalancer/configure/application/listeners.html

```diff
@@ -0,0 +1,59 @@
+<div class="container-fluid form-horizontal">
+  <div class="form-group">
+    <div class="col-md-12">
+      <table class="table table-condensed packed">
+        <thead>
+        <tr>
+          <th style="width:70px;">Protocol</th>
+          <th style="width:70px;">Port</th>
+          <th></th>
+          <th>Targets</th>
+          <th ng-if="ctrl.showSslCertificateNameField()">Certificates</th>
+          <th></th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr ng-repeat="listener in loadBalancer.listeners">
+          <td><select class="form-control input-sm" ng-model="listener.protocol"
+                      ng-change="ctrl.listenerProtocolChanged(listener)"
+                      ng-options="protocol for protocol in ['HTTP','HTTPS']"></select></td>
+          <td><input class="form-control input-sm" type="number" min="0" ng-model="listener.port"
+                     required/></td>
+          <td class="small" style="padding-top: 10px;">&rarr;</td>
+
+          <td>
+            <div ng-repeat="action in listener.defaultActions">
+              <select class="form-control input-sm" ng-model="action.targetGroupName"
+                      ng-options="targetGroup.name as targetGroup.name for targetGroup in loadBalancer.targetGroups" required></select>
+            </div>
+          </td>
+          <td>
+            <div ng-repeat="certificate in listener.certificates"
+                 style="width: 100%;display: flex;flex-direction: row;">
+              <select class="form-control input-sm inline-number"
+                      style="width:45px;"
+                      ng-model="certificate.type"
+                      ng-options="certificateType for certificateType in ['iam','acm']"></select>
+              <input class="form-control input-sm no-spel" style="display:inline-block;"
+                     type="text"
+                     ng-model="certificate.name"
+                     required/>
+            </div>
+          </td>
+          <td><a href class="sm-label" ng-click="ctrl.removeListener($index)"><span
+            class="glyphicon glyphicon-trash"></span></a></td>
+        </tr>
+        </tbody>
+        <tfoot>
+        <tr>
+          <td colspan="{{ctrl.showSslCertificateNameField() ? 7 : 5}}">
```

Comment:
This should probs be:

`<td colspan="{{ctrl.showSslCertificateNameField() ? 6 : 5}}">`